### DISCUSSION TITLE
Downloading track data

### DISCUSSION THREAD

**Discussion by celegans1234** [GMOD/jbrowse-components#3312] [Q&A]
Hi!
Could someone please help me with downloading a track data with annotations? This was possible in the previous JBrowse version. But in the current JBrowse, I am only able to download a FASTA file of a sequence. (I would like to then open the downloaded file using Geneious Prime.)
Thanks a lot!

**Comment by cmdcolin** [comment]
Currently we don't have the export functionality yet like in jbrowse 1, but it has been requested, and hopefully can be added to jbrowse 2! Similar thread https://github.com/GMOD/jbrowse-components/issues/3094

**Reply by cmdcolin** [reply]
This is just a random comment but if you can describe the geneious workflow that you use with the exported data I would be curious, especially if there are any ways jbrowse could make it easier

**Reply by celegans1234** [reply]
Hi, thanks a lot for responding!
I'm trying to get the sequence of unc-119 and its genomic location so that I can design appropriate primers for it. (This was straight-forward in the previous version of JBrowse as I could save the track-data and import it in Geneious...)
In general, for extracting a cDNA sequence the annotations of the intron and exon position of this gene would also be required.
Thanks a lot for helping out!!
